src/specapi.jl

```diff
@@ -401,6 +401,13 @@ function to_plot_object(ps::PlotSpec)
     return P((ps.args...,), copy(ps.kwargs))
 end
 
+function to_plot_object(ps::PlotSpec, p::PlotList)
+    P = plottype(ps)
```

Comment:
What would people think if, knowing the `default_theme` of `P()::Plot` here, we populate the missing attributes with their default values as Observables in `p::PlotList`, so that users can edit them post-facto and have them pass through like normal Observables?